cezo_fl/util/prepare_settings.py

```diff
@@ -0,0 +1,191 @@
+import torch
+import torch.nn as nn
+
+from peft import LoraConfig, get_peft_model
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+
+from cezo_fl.util import model_helpers
+from cezo_fl.models.cnn_fashion import CNN_FMNIST
+from cezo_fl.models.cnn_mnist import CNN_MNIST
+from cezo_fl.models.lenet import LeNet
+from cezo_fl.models.lstm import CharLSTM
+from cezo_fl.random_gradient_estimator import RandomGradientEstimator as RGE
+from cezo_fl.util.language_utils import LM_TEMPLATE_MAP, SUPPORTED_LLM, get_lm_loss
+from cezo_fl.util.metrics import accuracy
+
+
+def prepare_settings_underseed(args, device, server_or_client: str = "server"):
```

Comment:
I didn't move this function into settings because of this args, which is unstructured and can be anything. I would suggest to add another intermedia buffer between this function and args so that we don't need to worry about this function when refactor `args`. Wdyt? (not necessary in this PR)

please add more type annotation and comment after moving into modules.

Reply:
https://github.com/ZidongLiu/DeComFL/issues/64, created a issue.
I will make a dataclass for args in another PR

Reply:
which can be used for typing all args